superset/Dockerfile

```diff
@@ -59,9 +60,15 @@ RUN microdnf update \
         python-json-logger \
     && if [ ! -z "$AUTHLIB" ]; then pip install Authlib==${AUTHLIB}; fi
 
+    # Choosing correct binary with respect to architecture
+WORKDIR /stackable
```

Comment:
The working directory is changed here, what makes the apply_patches.sh script fail:

```
 => ERROR [superset-3_0_1 builder 6/7] RUN /patches/apply_patches.sh 3.0.1                                                                                                                0.2s
------
 > [superset-3_0_1 builder 6/7] RUN /patches/apply_patches.sh 3.0.1:
0.132 Patches directory 'patches/3.0.1' does not exist.
------
```

A quick solution would be to move the statsd_exporter section behind the patch section.

Reply:
done with https://github.com/stackabletech/docker-images/pull/534/commits/272e4e1ac9491302257e2e03077ea619edfae7e2